core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala

```diff
@@ -147,6 +148,14 @@ private[ui] class ExecutorTable(stageId: Int, stageAttemptId: Int, parent: Stage
                 {Utils.bytesToString(v.diskBytesSpilled)}
               </td>
             }}
+            <td>
+              {val logs = parent.executorsListener.executorToLogUrls.getOrElse(k, Map.empty)
+               if (logs.isEmpty) {
+                 "No Logs Found"
```

Comment:
The executors page currently leaves this blank when there are no logs, for consistency across the ui it may be better to remove this string.